src/osmf/js/osmf_flash.js

```diff
@@ -383,15 +408,15 @@
     this.destroy = function() {
       // Pause the video
       this.pause();
-
       // Reset the source
       this.setVideoUrl('');
 
       // Unsubscribe all events
+      
       this.unsubscribeAllEvents();
 
       // Pass destroy to flash plugin.
-      this.callToFlash("destroy");
+      this.callToFlash("destroy",null);
```

Comment:
if exactly one parameter is always expected, I would recommend having callToFlash turn undefined into null